src/main/java/org/broadinstitute/hellbender/utils/codecs/gencode/GencodeGtfCodec.java

```diff
@@ -63,7 +63,7 @@
     static final Logger logger = LogManager.getLogger(GencodeGtfCodec.class);
 
     public static final int GENCODE_GTF_MIN_VERSION_NUM_INCLUSIVE = 19;
-    public static final int GENCODE_GTF_MAX_VERSION_NUM_INCLUSIVE = 27;
+    public static final int GENCODE_GTF_MAX_VERSION_NUM_INCLUSIVE = 28;
```

Comment:
Based on our conversation earlier today, I think it makes sense to remove the `GENCODE_GTF_MAX_VERSION_NUM_INCLUSIVE` field altogether and just attempt to use the specified gencode.  

We can add in a warning / exception when an error occurs to investigate the Gencode version.

Reply:
Done